sanic/exceptions.py

```diff
@@ -266,3 +266,7 @@ def abort(status_code: int, message: Optional[Union[str, bytes]] = None):
     )
 
     raise SanicException(message=message, status_code=status_code)
+
+
+class ResponseException(SanicException):
```

Comment:
Do we need this? Why not `ServerError`?

Reply:
Originally, I thought we need a new type of `Exception` that we would need to ignore in `handle_exception`, but now `handle_exception` can detect the stage, so we don't need it anymore. 

Fixed.